dropwizard/service/src/main/java/org/apache/polaris/service/dropwizard/tracing/QuarkusTracingFilter.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.dropwizard.tracing;
+
+import io.opentelemetry.api.trace.Span;
+import io.quarkus.vertx.web.RouteFilter;
+import io.vertx.ext.web.RoutingContext;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import org.apache.polaris.service.dropwizard.logging.QuarkusLoggingMDCFilter;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@ApplicationScoped
+public class QuarkusTracingFilter {
+
+  public static final String REQUEST_ID_ATTRIBUTE = "polaris.request.id";
+  public static final String REALM_ID_ATTRIBUTE = "polaris.realm";
+
+  @ConfigProperty(name = "quarkus.otel.sdk.disabled")
+  boolean sdkDisabled;
+
+  @Inject QuarkusTelemetryConfiguration telemetryConfiguration;
+
+  @RouteFilter(QuarkusLoggingMDCFilter.PRIORITY - 1)
+  public void applySpanAttributes(RoutingContext rc) {
+    if (!sdkDisabled) {
+      Span span = Span.current();
+      telemetryConfiguration.spanAttributes().forEach(span::setAttribute);
```

Comment:
For global properties it is probably more efficient to use resource attributes as opposed to span attributes, but we can deal with that later.